swan-cern/values.yaml

```diff
@@ -258,154 +258,61 @@ hadoopTokenGenerator:
   image: gitlab-registry.cern.ch/swan/docker-images/hadoop-token-generator:v2.1.2
 
 fluentd:
-  containerRuntime: containerd
+  enabled: true
```

Comment:
Is this necessary? Isn't it enabled by default? I can't find this attribute [here](https://github.com/fluent/helm-charts/blob/main/charts/fluentd/values.yaml).

Reply:
It was being used to not include the fluentd config maps (with the fluentd configuration) if we ever decided to disable it. After further review, the option was removed, since fluentd is part of the SWAN cern infrastructure and shouldn't be disabled.